pkg/linode-bs/deploy/kubernetes/08-cm-get-linode-id.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: get-linode-id
+  namespace: kube-system
+  labels:
+    app: csi-linode
+data:
+  get-linode-id.sh: |-
+    #!/bin/bash -efu
+    id="$(kubectl get node/"${NODE_NAME}" -o jsonpath='{.metadata.annotations.lke\.linode\.com/linode-id}')"
+    if [[ ! -z "${id}" ]]; then
+      echo "Linode ID: "${id}"" 
+      echo -n ""${id}"" > /linode-info/linode-id
```

Comment:
Do the double quotes in this string need to be escaped?

Reply:
No, since I am not trying to print the quotes themselves. I am just quoting the variable and then quoting the string that gets echo'd out, although just the quotes around echo seem like they should be enough. The double quotes don't hurt anyone in this case, however.

Reply:
If you don't expect to print the inner double quotes, you should remove them. As is, this is three strings being concatenated together, as in the following example.

```
$ docker run -ti debian /bin/bash
root@a12bc3622668:/# a="hello"
root@a12bc3622668:/# echo -n ""${a}""
helloroot@a12bc3622668:/# echo -n "yes"${a}"ok"
yeshellookroot@a12bc3622668:/#
```

Reply:
Removed! 👍